code/src/main/webapp/CSS/style.css

```diff
@@ -1,13 +1,57 @@
-h1 {  
-  color: rgba(43, 141, 190, 1);
-  text-align: center;
+body, html {
+  height: 100%;
+  margin: 0;
+  color: #fefff2;
```

Comment:
This text isn't readable against the image (because the image has full-white too). 

I'd recommend finalizing the image so that we can move forward, and then using a scrim (or one of the other techniques from here:)

https://blog.iamsuleiman.com/techniques-to-display-text-overlay-background-images/

Reply:
done